ts/context.ts

```diff
@@ -799,7 +800,9 @@ export class QuickJSContext implements LowLevelJavascriptVm<QuickJSHandle>, Disp
 
       const fn = this.fnMap.get(fn_id)
       if (!fn) {
-        throw new Error(`QuickJSContext had no callback with id ${fn_id}`)
+        console.error('Could not find function', fn_id)
+        process.exit(1)
```

Comment:
It’s not appropriate to process.exit here.

Reply:
yes sorry, this was from a debugging pass. its been removed